config/can_config.h

```diff
@@ -48,6 +48,21 @@ typedef enum CANID {
 
   // GIMBTONOMY CAN IDS
   SET_NEOPIXEL = 0X784,
-  NEOPIXEL_ACK
+  NEOPIXEL_ACK,
+
+  // Science CAN IDs
+
+  SET_INDEXER_POS = 0x760,
```

Comment:
You might want to reorder some of these addresses to keep with the consistency. I think all you would have to do is move the neopixel CAN Id's to be after the CAN Id's you have created